src/components/Book.module.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&family=Roboto+Slab:wght@100;300;500;600;700&display=swap');
+
+.bookDiv {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  justify-content: center;
+  margin-bottom: 1rem;
+  width: 82vw;
+}
+
+.bookDetails {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 0 2rem 1.5rem;
+  flex-basis: 50%;
+}
+
+.category {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: #121212;
+}
+
+.author {
+  margin: 0.188rem 0 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.8rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
```

Comment:
- [ ] Again for repeating the same rules for [DRY CSS](https://speckyboy.com/good-bad-css-practices/) please combine them.

Reply:
Hi @ahmetbozaci thanks for that advice I will  make that change